### ISSUE TITLE
Add a MSSQL database in Azure used by e2e test

### ISSUE THREAD

**Issue by rickbrouwer** [kedacore/testing-infrastructure#160]
We need a MSSQL database in Azure for the e2e test for the MSSQL scaler to cover the azure-workload auth (https://github.com/kedacore/keda/pull/6161)

### Use-Case

We must check if the azure-workload auth works correctly. Therefore we need a MSSQL database in Azure to connect with a `WorkloadIdentityResource`.

### Specification

- [ ] Create a MSSQL database in MSSQL
- [ ] Add a azure WorkloadIdentity to connect

**Comment by JorTurFer** [comment]
I missed this issue, do you need help with it?

**Comment by rickbrouwer** [comment]
> I missed this issue, do you need help with it?

Yes, this is something i need help with. That would be very nice

**Comment by JorTurFer** [comment]
I've created the sql server with a federated login for KEDA's identity, so you should be able to use it with the connection values from these envs:
- TF_AZURE_SQL_SERVER_FQDN -> full qualified name
- TF_AZURE_SQL_SERVER_ADMIN_USERNAME -> admin username
- TF_AZURE_SQL_SERVER_ADMIN_PASSWORD -> admin password
- TF_AZURE_SQL_SERVER_DB_NAME -> database name

KEDA's identity has `db_owner` role in the database, so you should be able to test your code 

And soryy for the huge delay :(
